modelina-website/src/components/playground/options/GeneralOptions.tsx

```diff
@@ -64,15 +66,27 @@ class GeneralOptions extends React.Component<
     }
   }
 
+  
+
+
   render() {
     return (
       <ul className="flex flex-col">
         <h3 className="text-lg font-medium leading-6 text-gray-900">
           General options
         </h3>
-        <li>
-          <label className="flex items-center py-2 justify-between cursor-pointer">
-            <span className="mt-1 max-w-2xl text-sm text-gray-500">
+        <li className=' flex items-center mt-3'>
+          <InfoModal
+            html={`
+              Additional information about the output type
```

Comment:
How about instead of dangerously setting html we use children?

Reply:
This would also resolve the issue of font-family I guess it should be Inter inside the modal too.

Reply:
> This would also resolve the issue of font-family I guess it should be Inter inside the modal too.

I think that would be an overkill, as it will also require to parse html to jsx , which won't be possible without looping through the entire text, this could create performance bottleneck :crying_cat_face: 

well i purposely changed the font here `<div className=' px-3 py-2 text-sm font-mono'> ` to give it a more distinct look, it can be resolved by simply removing 'font-mono' from the tailwind class :smile:

Reply:
> How about instead of dangerously setting html we use children?

I agree that would definitely be better 👍 

Why do you see that as a bottleneck @SumantxD ? 🤔

Reply:
It would also make it easier to design the modal instead of it just being pure text 🙂

Reply:
> Why do you see that as a bottleneck @SumantxD ? thinking

my bad i misunderstood it with something else :face_with_spiral_eyes: 


> It would also make it easier to design the modal instead of it just being pure text slightly_smiling_face

I have fixed that :smile_cat: 

ps: @Shurtu-gal thx for the suggestion :smiley_cat: